File: javascript/returnBinary.js
    function returnBinary(n, str){
        if(n<2) return n+str;
        return returnBinary(Math.floor(n/2), n%2+str);
    }

    console.log(returnBinary(2, ""));
    console.log(returnBinary(5, ""))
    console.log(returnBinary(10, ""))

File: java/src/ReturnBinary.java
public class ReturnBinary {

    public static void main(String[] args) {
        System.out.println("\n");
        System.out.println(returnBinary(5, ""));
        System.out.println(returnBinary(8, ""));
        System.out.println(returnBinary(2, ""));
    }

    private static String returnBinary(int n, String str){
        if(n<2) return n+str;
        return returnBinary(n/2, n%2+str);
    }
}
